_validate/createJson.py

```diff
@@ -31,6 +32,10 @@ def getSha256(addonPath: str) -> str:
 	return sha256Addon
 
 
+def getCurrentTime() -> float:
+	return time.time() * 1000  # Milliseconds
```

Comment:
Use UTC time and make it an int

```suggestion
def getCurrentTime() -> int:
	return int(time.gmtime() * 1000)  # Milliseconds
```